tests/attacks/lucash-flip-end.mcd.expected

```diff
@@ -88,6 +88,7 @@
       ListItem ( LogNote ( ADMIN , Pot . cage ) )
       ListItem ( LogNote ( ADMIN , End . cage ) )
       ListItem ( LogNote ( ADMIN , End . cage "gold" ) )
+      ListItem ( TimeStep 3600 )
```

Comment:
This should probably be `ListItem ( TimeStep(3600,3600) )`

Reply:
Done